neo/EventHub.py

```diff
@@ -31,39 +42,72 @@ class SmartContractEvent:
     RUNTIME_NOTIFY = "SmartContract.RuntimeNotify"  # payload: object[]
     RUNTIME_LOG = "SmartContract.RuntimeLog"        # payload: bytes
 
-    # EXECUTION_INVOKE = "SmartContract.ExecutionInvoke"
-    # EXECUTION_SUCCESS = "SmartContract.ExecutionSuccess"
-    # EXECUTION_FAIL = "SmartContract.ExecutionFail"
+    EXECUTION_INVOKE = "SmartContract.ExecutionInvoke"
+    EXECUTION_SUCCESS = "SmartContract.ExecutionSuccess"
+    EXECUTION_FAIL = "SmartContract.ExecutionFail"
 
     event_type = None
     event_payload = None
     contract_hash = None
     block_number = None
     tx_hash = None
+    execution_success = None
 
-    def __init__(self, event_type, event_payload, contract_hash, block_number, tx_hash):
+    def __init__(self, event_type, event_payload, contract_hash, block_number, tx_hash, execution_success=False):
         self.event_type = event_type
         self.event_payload = event_payload
         self.contract_hash = contract_hash
         self.block_number = block_number
         self.tx_hash = tx_hash
+        self.execution_success = execution_success
 
     def __str__(self):
-        return "SmartContractEvent(event_type=%s, event_payload=%s, contract_hash=%s, block_number=%s, tx_hash=%s)" % (self.event_type, self.event_payload, self.contract_hash, self.block_number, self.tx_hash)
+        return "SmartContractEvent(event_type=%s, event_payload=%s, contract_hash=%s, block_number=%s, tx_hash=%s, execution_success=%s)" \
+               % (self.event_type, self.event_payload, self.contract_hash, self.block_number, self.tx_hash, self.execution_success)
 
 
 # Helper for easier dispatching of events from somewhere in the project
-def dispatch_smart_contract_event(event_type, event_payload, contract_hash, block_number, tx_hash):
-    sc_event = SmartContractEvent(event_type, event_payload, contract_hash, block_number, tx_hash)
+def dispatch_smart_contract_event(event_type,
+                                  event_payload,
+                                  contract_hash,
+                                  block_number,
+                                  tx_hash,
+                                  execution_success=False):
+
+    sc_event = SmartContractEvent(event_type, event_payload, contract_hash, block_number, tx_hash, execution_success)
     events.emit(event_type, sc_event)
 
 
 #
 # These handlers are only for temporary development and testing
 #
-@events.on("*")
-@events.on("*.*")
-def on_any_event(*args):
-    print("")
-    print("=EVENT: %s" % args)
-    print("")
+
+
+@events.on(SmartContractEvent.RUNTIME_LOG)
+def on_sc_log(*args):
```

Comment:
if you know these events always include a SmartContractEvent as argument, we could change the method signature from this:

    def on_sc_log(*args):
        if len(args) > 0 and type(args[0]) is SmartContractEvent:
            sc_event = args[0]

to

    def on_sc_log(sc_event):